src/Symfony/Component/Routing/Matcher/Dumper/PhpMatcherDumper.php

```diff
@@ -56,13 +55,11 @@ public function dump(array $options = array())
 
         // trailing slash support is only enabled if we know how to redirect the user
         $interfaces = class_implements($options['base_class']);
-        $supportsRedirections = isset($interfaces[RedirectableUrlMatcherInterface::class]);
```

Comment:
what about this distinction between supporting it or no ? Is it gone ?

Reply:
Yes it's gone, it fits no purpose.

Reply:
Oups, totally unrelated answer I had something else in mind :)
So no, it's not gone, it's here:
#28805